kernel/build-kernel.sh

```diff
@@ -0,0 +1,252 @@
+#!
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#---------------------------------------------------------------------
```

Comment:
Idea - you could define this description as a variable (similar to what I did for `license=` here: https://github.com/kata-containers/tests/blob/master/.ci/kata-doc-to-script.sh#L2). That way, it stays at the top of the file so will hopefully be read by developers, but it could also be displayed in `usage()`.